src/app/components/publiclists/teamlist/teamlist.component.html

```diff
@@ -43,7 +43,7 @@
                      #teamName="ngModel" [(ngModel)]="create_team['team_name']" required>
               <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
               <label for="name" [class.active]="isnameFocused" data-error="wrong" data-success="right">Team Name*</label>
-              <div class="wrn-msg text-highlight" *ngIf="teamName.invalid && (teamName.dirty || teamName.touched || createTeamForm.submitted)">Team Name Required</div>
+              <div class="wrn-msg text-highlight" *ngIf="validateTeamName && teamName.invalid && (teamName.dirty || teamName.touched || createTeamForm.submitted)">Team Name Required</div>
```

Comment:
As it is only a `required` validation then we can remove `teamName.invalid && (teamName.dirty || teamName.touched || createTeamForm.submitted` from the `if` condition and can use a single variable to toggle the `Team Name Required` message.

Make default of this variable to `false` and toggle this variable inside `createTeamSubmit` function on the `else` case when form is not valid if `team_name` is empty.

Also please change variable name to `isTeamNameRequired`

Please let me know if I'm not clear.

Reply:
I've done the changes you've requested, may I resolve this conversation?